features/weth/Weth.tsx

```diff
@@ -114,33 +132,199 @@ const Weth = () => {
+          </Typography>
+        </Box>
+        <Grid container spacing={3}>
+          <Grid item md={6} sm={6} xs={12}>
+            <Box style={{ height: "100%" }}>
+              <Container style={{ height: "100%" }}>
+                <Typography variant="h5">Your Wallet</Typography>
+                <BalanceElement>
+                  <IconAndNameContainer>
+                    <TokenIcon src={ETHEREUM_LOGO_URL} />
+                    <TokenName>ETH</TokenName>
+                  </IconAndNameContainer>
+                  <TokenBalance>Ξ {prettyEthBalance}</TokenBalance>
+                </BalanceElement>
+                <BalanceElement>
+                  <IconAndNameContainer>
+                    <TokenIcon src={ETHEREUM_LOGO_URL} />
+                    <TokenName>WETH</TokenName>
+                  </IconAndNameContainer>
+                  <TokenBalance>Ξ {prettyWethBalance}</TokenBalance>
+                </BalanceElement>
+              </Container>
+            </Box>
+          </Grid>
+          <Grid item md={6} sm={6} xs={12}>
+            <Box>
+              <Container>
+                <Typography variant="h5">Wrap & Unwrap</Typography>
+                <InputElement>
+                  <form
+                    style={{ width: "100%" }}
+                    noValidate
+                    autoComplete="off"
+                    onSubmit={(e) => handleClick(e, ACTION_TYPE.WRAP)}
+                  >
+                    <TextField
+                      fullWidth
+                      type="number"
+                      label="ETH"
+                      value={ethAmount}
+                      onChange={(e) => setEthAmount(e.target.value)}
+                      variant="outlined"
+                      inputProps={{ min: "0", max: ethBalance as string }}
+                      error={ethAmountAboveBalance}
+                      helperText={
+                        ethAmountAboveBalance && `Your ETH balance is too low`
+                      }
+                      InputLabelProps={{
+                        shrink: true,
+                      }}
+                      InputProps={{
+                        endAdornment: (
+                          <InputAdornment position="end">
+                            <Button
+                              type="submit"
+                              variant="contained"
+                              disabled={
+                                ethAmountAboveBalance || ethAmountToWrap <= 0
+                              }
+                            >
+                              Wrap
+                            </Button>
+                          </InputAdornment>
+                        ),
+                      }}
+                    />
+                  </form>
+                </InputElement>
+                <InputElement>
+                  <form
+                    style={{ width: "100%" }}
```

Comment:
there is a `fullwidth` prop that you should be able to apply. does that not work for the `form` ?

Reply:
Yeah its not a property of <form>. I think its a Material UI prop right and this is just a plain form